src/app/legacy/containers/CpsRecommendations/RecommendationsPromoList/index.jsx

```diff
@@ -25,7 +25,7 @@ const RecommendationsPromoListItem = forwardRef(
   // 004_brasil_recommendations_experiment
   ({ item, index, service, optimizely }, forwardedRef) => {
     const eventTrackingData =
-      service === 'portuguese'
+      service === 'portuguese' || service === 'mundo'
```

Comment:
This might not stick depending on wether we pass new values to the recs tracking for optimizely and ati